src/openrct2/paint/Paint.h

```diff
@@ -82,12 +72,28 @@ struct paint_string_struct
     uint8_t* y_offsets;
 };
 
-union paint_entry
+struct paint_entry
```

Comment:
What is the idea behind the changes to `paint_entry`?

Reply:
You can't use a complex type in a union

Reply:
ImageId has a default constructor, you can not define structs / classes with default constructors inside a union.